Client/Frontend/Reader/ReaderModeStyleViewModel.swift

```diff
@@ -0,0 +1,41 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+// MARK: - ReaderModeStyleViewModel
+
+// TODO: move all logic from VC in this view model
```

Comment:
```suggestion
```

can we get rid of todo or do we need to actually move some logic 😅

Reply:
We need to actually move the logic. Same type of answer as last one. I created the View models since the VC didn't contain any. It's better to start creating them than waiting I believe. Not ideal since this is an inbetween state, but then I prefer that than not making any improvements

Reply:
One solution for TODOs is I can create tech debt tasks and remove the TODOs in the code in the meantime. Let me know if this is a better solution.

Reply:
Oh hmm, this can be easily missed as we don't know if anyone will come back to this logic anytime soon. Basically trying to avoid ToDos that won't get resolved ha

@lmarceau would it be possible to add a ticket or a bit longer explanatory note so we actually address this in the future, what do you think? 🤔 I am open to suggestion(s).

Reply:
Task: https://github.com/mozilla-mobile/firefox-ios/issues/9976